src/5e-SRD-Monsters.json

```diff
@@ -14819,10 +15348,6 @@
             "damage_dice": "3d8"
           }
         ]
-      },
-      {
-        "name": "Variant: Rope of Entanglement",
-        "desc": "Some erinyes carry a rope of entanglement (detailed in the Dungeon Master's Guide). When such an erinyes uses its Multiattack, the erinyes can use the rope in place of two of the attacks."
```

Comment:
Should this be removed?

Reply:
I can leave it it if you want, but it doesn't come from the SRD.

Reply:
If it's not in the SRD, take it out.